### ISSUE TITLE
State trie does not contain state root

### ISSUE THREAD

**Issue by lixinfenggh** [ethereumjs/ethereumjs-monorepo#2436]
I try to build a custom block chain. First, I start node1, it works well.
![image](https://user-images.githubusercontent.com/15077525/204476082-c3a804de-09fb-44c1-bb82-3e08d070ec34.png)

Then I start node2.It seems to work on its own first. Then an error was thrown while it tried to import blocks from node1.
![image](https://user-images.githubusercontent.com/15077525/204477159-118515ab-914e-44e7-9226-f5114ec66cc0.png)

Here is my command and config file.

```
    "client:start:node1": "ts-node bin/cli.ts  --datadir=test/node1 --customChain=test/chainConfig.json --customGenesisState=test/customGenesis.json --bootnodes --mine=true --minerCoinbase=0x2fd74ee9bD5e82526886536f4cFF194e11cB3726  --unlock=test/node1/privatekey.txt  --port=30306 --rpcport=8599 --rpcEnginePort=8551  --syncmode=full --multiaddrs=127.0.0.1:50580",
    "client:start:node2": "ts-node bin/cli.ts --datadir=test/node2 --customChain=test/chainConfig.json --customGenesisState=test/customGenesis.json  --bootnodes=enode://0ce4e7a4c5fb05088ffda485d8cab50aab85e55add69b0b224bd3da5f50d93605b07bbeab91521e1b1ecd31de99507e24569937555c5142474f8c018c7775e55@127.0.0.1:30306 --unlock=test/node2/privatekey.txt  --port=30307 --rpcport=8600 --rpcEnginePort=8552  --syncmode=full --multiaddrs=127.0.0.1:50581 --mine=true --minerCoinbase=0xc53f46eb68CbF78cAAAB187F8AD435bCC82E3B48",
```
chainConfig.json
``` 
{
  "name": "mq-test-net",
  "chainId": "233333",
  "networkId": "233333",
  "consensus": {
    "type": "pow",
    "algorithm": "ethash"
  },
  "comment": "Test network",
  "genesis": {
    "difficulty": 1,
    "gasLimit": 100000,
    "nonce": "0xbb00000000000000"
  },
  "hardforks": [
    {
      "name": "chainstart",
      "block": 0
    }
  ]
}
```
customGenesis.json
``` 
{
    "0x2fd74ee9bD5e82526886536f4cFF194e11cB3726": "0x3000000000",
    "0xc53f46eb68CbF78cAAAB187F8AD435bCC82E3B48": "0x4000000000",
    "0x2b17715f734Ce8dae3293e8DecC804aCfC5Ae28A": "0x5000000000"
}
```

**Comment by acolytec3** [comment]
Sorry for the slow reply.  I tried to recreate this issue and wasn't able to. I did find a separate bug in our `common` package related to the use of the `customChain` cli parameter I will research further but once I resolved that locally, I was able to start up two clients using your CLI parameters and chain config/state (adjusting for local file names and switching out private keys/coinbases).  Is this still an issue?
